apps/web/app/[locale]/dashboard/team-dashboard/components/team-stats-table.tsx

```
@@ -0,0 +1,91 @@
+'use client';
+
+import { Avatar, AvatarFallback, AvatarImage } from '@/components/ui/avatar';
+import { Button } from '@/components/ui/button';
+import { Table, TableBody, TableCell, TableHead, TableHeader, TableRow } from '@/components/ui/table';
+import { ChevronLeft, ChevronRight, ChevronsLeft, ChevronsRight } from 'lucide-react';
+import { members } from '../data/mock-data';
+
+const getProgressColor = (activityLevel: string) => {
+	const level = parseInt(activityLevel, 10);
+	if (isNaN(level) || level < 0) return 'bg-gray-300';
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)